mongodb/configure.sls

```diff
@@ -1,4 +1,5 @@
 {% from "mongodb/map.jinja" import mongodb with context %}
+{% from "mongodb/map.jinja" import vagrant_host with context %}
```

Comment:
This works too, but you can also just add it to the first line. e.g. `from "mongodb/map.jinja" import mongodb, vagrant_host with context`